openfisca_us/entities.py

```diff
@@ -32,6 +15,7 @@
             doc="A member of the household",
         ),
     ],
+    containing_entities=["spm_unit"],
```

Comment:
I don't think the set of `containing_entities` arguments here consistently represents the hierarchy:

`person` -> `tax_unit` -> `family` -> `spm_unit` -> `household`

Should this include just the immediate predecessor in the hierarchy? SPMUnit having both family and household seems off.

Reply:
I mean, I should know which is the correct encapsulation, but in https://github.com/PolicyEngine/openfisca-us/issues/233 `spm_unit` and `family` are switched.

Reply:
Sorry - yes #233 was wrong (edited to fix). See https://www.census.gov/content/dam/Census/library/publications/2021/demo/p60-275.pdf#page=3

![image](https://user-images.githubusercontent.com/6076111/140652650-281ecf2d-2f13-49fc-97a1-ae7eee3098fd.png)